crates/tauri-plugin-service/android/src/main/AndroidManifest.xml

```diff
@@ -1,13 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
-  <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
```

Comment:
Moved these android manifest settings from tauri-plugin-service to the service library itself. Android building is smart enough to merge together the manifests of dependencies for the final application manifest.